weasel/cli/clone.py

```diff
@@ -53,7 +54,15 @@ def project_clone_cli(
         if not git_repo_branch_exists(repo, branch):
             msg.fail(f"repo: {repo} (branch: {branch}) does not exist.", exits=1)
     assert isinstance(branch, str)
-    project_clone(name, dest, repo=repo, branch=branch, sparse_checkout=sparse_checkout)
+    parent_command = _get_parent_command(ctx)
```

Comment:
`parent_command` would be e. g. `spacy` or `weasel`, right?

Reply:
Not exactly, the parent commands are either `weasel` or `spacy project`, and then sometimes with `python -m`.